locust/test/test_main.py

```diff
@@ -62,11 +67,16 @@ def web_interface_ready(port: int) -> bool:
 
 
 def all_users_spawned(proc: subprocess.Popen, output: list[str]) -> bool:
+    if proc.stderr is None:
+        raise ValueError("proc.stderr is None, cannot read from stderr")
```

Comment:
does this happen under any kind of normal circumstances? We don't need to check for things that should never, and this error message doesnt add much over the `AttributeError NoneType has no attribute 'readline'` that you would otherwise get anyway.